### PR TITLE
Upgrade to newer fork of hugo-kiera theme

### PR DESCRIPTION
**Pull request by DavidSpickett**
This fixes many issues when trying to build the site with the latest Hugo. The updated theme lists a minimum of v0.128.0 and the site builds fine with that too.

https://github.com/avianto/hugo-kiera is unmaintainted and https://themes.gohugo.io/themes/hugo-kiera/ lists https://github.com/funkydan2/hugo-kiera as the active fork of it.

So I have imported that at commit
https://github.com/funkydan2/hugo-kiera/commit/5676dfaf565148f472d6a35c1f244d1199454a49.

I have made no changes to anything in hugo-kiera/ myself.

I have also moved some things in our toml file:
* pagination has a new name
* author now needs to be in params.author

And updated some of our docs to tell people how to get the correct version of hugo.

### CODE REVIEW

**Comment by asl** [comment]
@DavidSpickett I think we'd also need to bump Hugo version at https://github.com/llvm/llvm-blog-www/blob/main/.github/workflows/main.yaml

File: themes/hugo-kiera/theme.toml [hunk 1]
```diff
@@ -5,7 +5,7 @@ description = "Hugo Theme for creative and technical writing"
 homepage = "https://github.com/funkydan2/hugo-kiera"
 tags = ["blog","minimal","responsive"]
 features = ["blog","disqus","google-analytics","font-awesome"]
-min_version = "0.55.0"
+min_version = "0.128.0"
```
Comment (DavidSpickett) [inline themes/hugo-kiera/theme.toml:8]: Can we find out what's used for the live site?

Comment (DavidSpickett) [reply themes/hugo-kiera/theme.toml:8]: Nice, I've done that.

**Comment by DavidSpickett** [comment]
v0.128.0 works for me locally.

**Comment by asl** [comment]
Cool, thanks a lot!

**Pull request merged**